components/02-molecules/quick-links/_yds-quick-links--links.twig

```diff
@@ -1,10 +1,15 @@
 {% set quick_links__base_class = 'quick-links' %}
 
+{% if quick_links__variation == 'subtle' %}
```

Comment:
Just to clarify on this one - this is component-based theming, not dictated by the global theme? So, if someone selects New Haven Green, these links will always be the darker blue?

Reply:
@codechefmarc Ah, yes, thank you for asking! 

The quick links `subtle` variation buttons don't currently change color based on global theme selection. The `promotional` variation, _does_, so maybe the buttons on `subtle` should, as well? I'll ask Ashley when we create this variation as a separate component. 

This component variation is going to be removed and moved into its own specific component. I'd like to wait until we build that. I'm silly for fixing the issue I did, here, but I wanted the layout piece as solid as possible for when we move the styles over to the new component.